src/js/planting.js

```diff
@@ -6,7 +6,7 @@ import SessionDataModel from 'session-data';
 import ManifestoDataModel from 'manifesto-data';
 import Main from 'module/main/main';
 import Plant from 'module/plant/plant';
-import Toolbox from 'module/toolbox/toolbox';
+import Sidebar from './module/toolbox/sidebar';
```

Comment:
In most cases we should use relative path (with `.`). Without dot means that we want import library from node_modules or different predefined location.